docs/team/community-strategy.md

```diff
@@ -0,0 +1,24 @@
+# Community Strategy
+
+This page contains information regarding activities undertaken and community strategies developed by the Community Strategic Lead.
+[Sarah Gibson](https://github.com/sgibson91) is currently serving in this role as of August 2022, for a period of two years.
+
+This role and related activities are funded by a CZI EOSS DEI grant, administered by NumFOCUS.
+
+```{admonition} Work in Progress
+The work represented on this page is under active development and we will update information as soon as we are able to as we iterate.
+```
+
+Some information is being shared in [Issue #536](https://github.com/jupyterhub/team-compass/issues/536), which will be migrated here over time.
+Additionally, you can find information on other related activities via the [![csl-label](https://img.shields.io/github/labels/jupyterhub/team-compass/community-strategic-lead)](https://github.com/jupyterhub/team-compass/issues?q=label%3Acommunity-strategic-lead) label.
+
+## Community Strategy Updates
+
+### 2022-10-07: 2 month strategy so far
+
+One strategy is to develop onboarding pathways through Outreachy,then into Contributor in Residence, and finally as a fully-fledged community member.
```

Comment:
Let's flesh this out in a future update - we can brainstorm via these updates, and then encode things in the team compass via more focused PRs